src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/DateHistogramParser.java

```diff
@@ -105,10 +104,8 @@ public AggregatorFactory parse(String aggregationName, XContentParser parser, Se
                     preZone = DateTimeZone.forID(parser.text());
                 } else if ("post_zone".equals(currentFieldName) || "postZone".equals(currentFieldName)) {
                     postZone = DateTimeZone.forID(parser.text());
-                } else if ("pre_offset".equals(currentFieldName) || "preOffset".equals(currentFieldName)) {
-                    preOffset = parseOffset(parser.text());
-                } else if ("post_offset".equals(currentFieldName) || "postOffset".equals(currentFieldName)) {
-                    postOffset = parseOffset(parser.text());
+                } else if ("offset".equals(currentFieldName)) {
```

Comment:
do we need to keep the old post / pre code to make this BWC. I wonder if we should do this since it's a REST api? 
another way of doing this is to remove it here and add the BWC code to 1.5 and deprecated `pre / post` there?